sdk/appconfiguration/tests.yml

```diff
@@ -7,4 +7,3 @@ jobs:
       EnvVars:
         AZURE_TEST_MODE: RECORD
         AZURE_APPCONFIG_CONNECTION_STRING: $(java-azconfig-test-connection-string)
```

Comment:
why remove log level altogether? It I'd expect at least AZURE_LOG_LEVEL = 4 for warnings. It's hard to diagnose errors when the logs are blank without rerunning them yourself.

Reply:
OK. I will add AZURE_LOG_LEVEL = 3 in the next PR: https://github.com/Azure/azure-sdk-for-java/pull/6572